import java.util.*;
class Second
{
public static void main(String args[])
{
Scanner s=new Scanner(System.in);
int n=s.nextInt();
try
{
System.out.println("satish");
System.out.println(10/0);
System.out.println("satishs");
}
catch(ArithmeticException e)
{
System.out.println(10/2);
}
}
}

//If the exception raised in try block the remaining code of try block is not executed
